.browser

```diff
@@ -0,0 +1 @@
+chrome@115.0.5763.0
```

Comment:
Not a blocker:

Instead of creating an arbitrary `.browser`, let's stick to a widely used convention like `.envrc` or `.env` (or similar):

```
$ cat ~/.envrc:
BROWSER=chrome@115.0.5763.0
```

Reply:
this is not really an env variable so I am not sure it needs to be in an env file.